org.pegasus_frontend.Pegasus.yml

```diff
@@ -25,7 +25,12 @@ modules:
       - INSTALL_ICONDIR=/app/share/icons/hicolor
       - INSTALL_DESKTOPDIR=/app/share/applications
       - INSTALL_APPSTREAMDIR=/app/share/metainfo
+    post-install:
+      - |
+        last_version=$(grep -oPm1 '(?<=version=").*(?=" />)' /app/share/metainfo/org.pegasus_frontend.Pegasus.metainfo.xml)
+        sed -i "s|<releases>|<releases>\n    <release date=\"$(git log -1 --format=%ci | awk '{print $1}')\" version=\"$last_version $(git rev-parse --short HEAD)\" \/>|" \
+        /app/share/metainfo/org.pegasus_frontend.Pegasus.metainfo.xml
     sources:
       - type: git
         url: https://github.com/mmatyas/pegasus-frontend.git
-        branch: release_alpha15
+        commit: 29ca7047018a2f45b42743b1facdaf21ab537e08
```

Comment:
I'd like to keep a `branch` here, in case someone wants to build this locally. You can of course replace this in the workflow.

Reply:
This may be possible but this will yield to unreproducible builds.

Reply:
We could add a comment, like `# branch: master`, as an example. Still, if one want to test a feature, (s)he will likely have to change the URL too to point to its fork.

Reply:
Why? To me it seems you'd just need to edit the `sed -i "s/  commit: .*/  commit: $hash/" org.pegasus_frontend.Pegasus.yml` from `commit` to `branch`. Am I missing something?

Reply:
But then the GitHub Actions commits to this repository, a commit will still be present at the end in master

Reply:
The build is not done in this GitHub Action CI, it just commits to master, which then triggers later a build in the flathub CI.

Reply:
Oh right, forgot about that. It's fine then.